### PR TITLE
latest ADF alpha

### PR DESCRIPTION
**Pull request by DenysVuika**
## PR Checklist
Please check if your PR fulfills the following requirements:

```
- [ ] The commit message follows our guidelines: https://github.com/Alfresco/alfresco-content-app/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
```

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Application / Infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: e2e/configs.ts [hunk 1]
```diff
@@ -42,6 +42,10 @@ export const ADMIN_FULL_NAME = 'Administrator';
 
 export const E2E_ROOT_PATH = __dirname;
 
+// Dates
+export const DATE_FORMAT = 'MMM DD, YYYY';
+export const DATE_TIME_FORMAT = 'MMM D, YYYY, h:mm';
```
Comment (suzanadirla) [inline e2e/configs.ts]: let's change the hour format to H:mm and also in the app.config - to have a 24h time format and avoid confusions

Comment (suzanadirla) [reply e2e/configs.ts]: or the correct format supported by moment -here

Comment (DenysVuika) [reply e2e/configs.ts]: this format is used in metadata tests that do not support app.config yet

Comment (suzanadirla) [reply e2e/configs.ts]: ok, understood

**Pull request merged**